### DISCUSSION TITLE
Issue with opening app on MacBook

### DISCUSSION THREAD

**Discussion by binkleswitha45** [redeclipse/base#1149] [Help and Support] [answered]
Hello I've been having issues trying to open Red Eclipse on my Macbook Air. Whats been happening is that when I try to open the game, my terminal comes up as well as the game, but then the game closes down and the terminal starts downloading the games files. It goes for a little while, and then it stops, asking for me to put in my administration password to continue installing the game. This is where I'm confused, because terminal on the mac won't let me type inside the terminal to open the game. Any help would be appreciated, thank you

**Comment by qreeves** [comment, marked as answer]
Generally, the updater only asks for the administrator password if the game is located in a non-user writable folder.

- What version of macOS are you running?
- What actions are you taking to download, install, and run the game?
- Where are you putting the Red Eclipse app file?
- Can you provide a screenshot of where you get stuck?
